default.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "group:recommended",
+    ":separateMultipleMajorReleases",
+    ":pinDevDependencies",
+    "github>smg-automotive/renovate-config:groupDevDependencies",
+    "github>smg-automotive/renovate-config:nodeUpdate",
+    "github>smg-automotive/renovate-config:dockerOS",
+    "github>smg-automotive/renovate-config:internalPackageUpdate"
+  ],
+  "reviewers": [
+    "team:frontend"
+  ],
+  "node": {
+    "supportPolicy": [
+      "lts_latest"
+    ]
+  }
```

Comment:
Let's force semantic commits. Auto-detection is a bit wonky and, especially on fresh repositories, does not always work well (check the commit history of the components package). This can lead to some issues in the packages so let's proactively prevent it.